src/params.rs

```diff
@@ -461,23 +461,23 @@ mod tests {
         assert_eq!(
             Ok(Params {
                 from: os("foo"),
-                to: os("/dev/stdin"),
+                to: os("-"),
                 ..Default::default()
             }),
             parse_params([os("diff"), os("foo"), os("-")].iter().cloned())
         );
         assert_eq!(
             Ok(Params {
-                from: os("/dev/stdin"),
+                from: os("-"),
                 to: os("bar"),
                 ..Default::default()
             }),
             parse_params([os("diff"), os("-"), os("bar")].iter().cloned())
         );
         assert_eq!(
             Ok(Params {
-                from: os("/dev/stdin"),
-                to: os("/dev/stdin"),
+                from: os("-"),
+                to: os("-"),
```

Comment:
Please keep at least one test that uses `/dev/stdin`. I think the right way to mark a test as "unix-only" is `#[cfg(unix)]` right before `fn`.

Reply:
Those tests do not actually exercise reading from stdin, only the parsing of command-line parameters. I don't see much added value to verifying that `/dev/stdin` doesn't get rewritten.

However I can add an integration test that exercises reading from `/dev/stdin`.